[PyTorch] AOTI: add CPU fast path in aoti_torch_empty_strided

swolchok opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #110892
* #110909
* __->__ #110877
* #110876

This seems to reduce benchmark time by 15-20%. Supersedes D49835545.

Differential Revision: [D49974460](https://our.internmc.facebook.com/intern/diff/D49974460/)

swolchok committed:
[PyTorch] AOTI: add CPU fast path in aoti_torch_empty_strided

This seems to reduce benchmark time by 15-20%. Supersedes D49835545.

Differential Revision: [D49974460](https://our.internmc.facebook.com/intern/diff/D49974460/)

[ghstack-poisoned]

chenyang78 reviewed (approved):
LGTM. Thanks!

swolchok committed:
Update on "[PyTorch] AOTI: add CPU fast path in aoti_torch_empty_strided"


This seems to reduce benchmark time by 15-20%. Supersedes D49835545.

Differential Revision: [D49974460](https://our.internmc.facebook.com/intern/diff/D49974460/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] AOTI: add CPU fast path in aoti_torch_empty_strided"


This seems to reduce benchmark time by 15-20%. Supersedes D49835545.

Differential Revision: [D49974460](https://our.internmc.facebook.com/intern/diff/D49974460/)

[ghstack-poisoned]

desertfire reviewed (approved):
More like a note to myself: we should do the same for other ops in this file

swolchok committed:
use V.graph._shape_env on "[PyTorch] AOTI: add CPU fast path in aoti_torch_empty_strided"


This seems to reduce benchmark time by 15-20%. Supersedes D49835545.

Differential Revision: [D49974460](https://our.internmc.facebook.com/intern/diff/D49974460/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] AOTI: add CPU fast path in aoti_torch_empty_strided"


This seems to reduce benchmark time by 15-20%. Supersedes D49835545.

Differential Revision: [D49974460](https://our.internmc.facebook.com/intern/diff/D49974460/)

[ghstack-poisoned]

swolchok commented:
@pytorchbot merge
